client/state/data-layer/wpcom/account-recovery/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+import {
+	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+} from 'state/action-types';
+
+export const requestResetOptions = ( { dispatch }, { userData } ) => (
+	wpcom.undocumented().accountRecoveryReset( userData ).getResetOptions()
```

Comment:
Sounds good. Removing a layer of indirectness seems doing no harm in this case. Updated by c08f04f66a696d65a5e7173c12b0f7bdff8f4133.